notifications.bs

```diff
@@ -767,13 +769,15 @@ method steps are:
  <a lt="create a PermissionStatus">creating a `PermissionStatus`</a> for
  <var>permissionDescriptor</var>.
 
- <li><p>Let <var>promise</var> be <a for=/>a new promise</a> in <a>this</a>'s <a>relevant Realm</a>.
-
  <li>
   <p>Run these steps <a>in parallel</a>:
 
   <ol>
-   <li><p>Run the <a for="powerful feature">permission query algorithm</a> with
+   <li><p>If <var>global</var>'s associated <a>environment settings object</a> is not a
+   <a>secure context</a>, return the result of <a>resolving a permission request promise</a> with
```

Comment:
I think you need to keep the existing setup here because you are returning a promise from in parallel steps, but that does not end up returning anything to the initial caller. It returns it into the in parallel void. 😊